woodwork/tests/test_utils.py

```diff
@@ -204,7 +210,7 @@ def test_read_csv_with_pandas_params(sample_df_pandas, tmpdir):
     sample_df_pandas.to_csv(filepath, index=False)
     nrows = 2
 
-    df_from_csv = ww.read_csv(filepath=filepath, nrows=nrows)
+    df_from_csv = ww.read_csv(filepath=filepath, nrows=nrows, dtype={'age': 'Int64', 'is_registered': 'boolean'})
```

Comment:
yay kwargs letting us get this behavior without making any changes to `read_csv`!